quadriga/src/main/java/edu/asu/diging/quadriga/network/model/Network.java

```diff
@@ -0,0 +1,75 @@
+package edu.asu.diging.quadriga.network.model;
+
+public class Network {
```

Comment:
the data model does not seem complete. Where is the information about the subject, predicate, objects? There are just ids, but it needs all the data.